setup.py

```diff
@@ -67,15 +67,16 @@
     "python-socketio~=3.0",
     "pydot~=1.4",
     "keras-applications==1.0.6",
-    "keras-preprocessing==1.0.5"
+    "keras-preprocessing==1.0.5",
+    "sklearn-crfsuite~=0.3.6",
```

Comment:
rasa_nlu >= 0.15.0a2

Reply:
same for requirements.txt

Reply:
requirements is already on `rasa_nlu~=0.15.0a4`